### PR TITLE
Exercise branch3

### PR DESCRIPTION
**Pull request by secretpants886**

### CODE REVIEW

**Review by eloivswild — APPROVED** [review]

**Review by racousin — CHANGES_REQUESTED** [review]
remove this file Zhan_XU/module1/user
 you already did it well in https://github.com/racousin/data_science_practice_2024/pull/33/files

File: secretpants886/module2/.DS_Store
Comment (racousin) [inline secretpants886/module2/.DS_Store:1]: remove .DsStore

**Review by racousin — CHANGES_REQUESTED** [review]

File: secretpants886/module2/.DS_Store
Comment (secretpants886) [reply secretpants886/module2/.DS_Store:1]: It's done sir.

**Review by racousin — APPROVED** [review]
Good work!

**Pull request merged**